### DISCUSSION TITLE
<title>I can't access my GitHub account

### DISCUSSION THREAD

**Discussion by husanIbragimov** [community/community#84946] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I don't know what did i do my GitHub account. GitHub doesn't sent verification code to my email. I reset Fedora OS to my Computer then I can't access. I don't remember which is function access sent me code but neither doesn't work. Then I don't have Recovery code file. How to access my GitHub account, please show way. I try many times

p.s. But  I know my GitHub password and username.

**Comment by ibra-kdbra** [comment, marked as answer]
- **Recovery Codes**: If you saved any recovery codes, you can use one to regain access to your account.(but you don't have any unfortunately) 
 
- **SSH Key**: If you have an SSH key assigned to your account, you can verify it by running a command on your computer where you have the SSH key. The output can be sent to GitHub support to disable your 2FA.
 
- **Verified Device**, **SSH Token**, or **Personal Access Token**: If you know your password but don’t have the 2FA credentials or recovery codes, you can have a one-time password sent to your verified email address to begin the verification process and regain access to your account.

- **Contact GitHub Support**: If none of the above methods work, you may need to contact GitHub support for further assistance. Please note that for security reasons, GitHub Support will not be able to restore access to accounts with two-factor authentication enabled if you lose your two-factor authentication credentials or lose access to your account recovery methods.

**Reply by husanIbragimov** [reply]
I don't think I got github-recovery-code.txt file because I installed Ubuntu in my computer. So now i don't know how to get it.


I don't get SSH KEY.

How do I use my personal access token?  I only used the access token to push my project to github.

Nowadays I am waiting support's answer

**Reply by ibra-kdbra** [reply]
1-If you are on _**Ubuntu OS**_, try this command:
```bash
find / -iname FILENAME
```

2- For the **SSH** key you should have added one to your system when connecting to the version control (Github, GitLab, etc...).


3- **Personal access token** can be used to inform the GitHub server that you are accessing without the need for confirmation(as you stated you don't have it).

After recovering the account if the support community answered quickly you need to add all these things to avoid this trouble later on.

Good luck and best wishes.

**Comment by ghostinhershell** [comment]
Hi @ibragimovxusan, thanks so much for reaching out in our community. 

We do have a support article that documents next steps if you lose access to your two-factor authentication credentials: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). I think this section, specifically, should be helpful: [Requesting help with two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#requesting-help-with-two-factor-authentication).

If that doesn't help, you can submit a support ticket via our [Support page](https://support.github.com/contact).

**Comment by CyberLordX7** [comment]
I have lost access to my 7 Years Worth GitHub Account. 
Please i really need help regarding getting my account back. Please GitHub Help me.

**Reply by ibra-kdbra** [reply]
**Contact** their support.

**Reply by CyberLordX7** [reply]
Please could you guide me on how to go about this. I have sent multiple emails to support@gmail.com but no response from them. the funny part is they didn't even email me the reason why my account was disabled. this is a 7+ years account linked with other important accounts. it's so unfair. Please Can you help me out.

**Reply by ibra-kdbra** [reply]
I can't help, I have a similar problem with GitHub account which was suspended for no obvious reason, the support didn't contact me either, just saved the ticket, what did I except when Microsoft bought this company; so...

**Comment by mohamedayoub97** [comment]


**Reply by ibra-kdbra** [reply]
What a lousy answer copied from  GPT...

At least help others, not putting the same prompt waiting an answer, you didn't put the effort for changing the same keywords.

**Reply by CyberLordX7** [reply]
Thank you for the feedback @mohamedayoub97. I want to drop this message on github community to see if anyone there can help me out: I recently discovered that my GitHub account, which I’ve been using for over 7 years, was disabled without any prior notification or explanation. I’ve sent multiple emails to support@gmail.com but haven’t received any response. What’s even more concerning is that I wasn’t informed of the reason for the account being disabled.

This account is linked to several important services, making the situation quite distressing. I feel this is very unfair and would deeply appreciate any guidance or assistance in resolving this matter.

Thank you in advance for your help.

**Reply by mohamedayoub97** [reply]
Hey, I know you are frustrated, but that is no reason to be nasty. I am trying a new AI bot based on Llama 3.1, and that is why my response wasn't as personalized. I'll make sure to improve in the future. But the answer given was correct, so please take it easy. Thanks for understanding.

**Reply by ibra-kdbra** [reply]
Oh come on, if the issue can be solved with AI, the user wouldn't ask in this forum.

I am not frustrated, it's just low way to answer the users.
